packages/card/src/__demo__/examples/card.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import { createStyledComponent } from '@mineral-ui/component-utils';
+import Card from '../../Card';
+import CardBlock from '../../CardBlock';
+import CardTitle from '../../CardTitle';
+
+const Root = createStyledComponent('div', {
```

Comment:
Not sure that we should keep all of these in here, as Cards automatically stretch to fill the available width and this makes it seem otherwise.